main.go

```diff
@@ -34,6 +34,35 @@ func main() {
 		version.PrintVersionAndExit()
 	}
 
+	if s.AutoDiscoverBaseArn {
+		if s.BaseRoleARN != "" {
+			log.Fatal("--auto-discover-base-arn can't be used if --base-role-arn is specified")
+		}
+		if s.DefaultIAMRole == "" {
+			log.Fatal("You need to specify the default IAM role, use --auto-discover-default-role to autodiscover it")
```

Comment:
Is just a warning, if DefaultIamRole is blank print that is a required argument and suggest the use of --auto-discover-default-role